sda/cmd/api/api.md

```diff
@@ -19,6 +19,12 @@ Endpoints:
   [{"inboxPath":"requester_demo.org/data/file1.c4gh","fileStatus":"uploaded","createAt":"2023-11-13T10:12:43.144242Z"}] 
   ```
 
+    It is possible to limit the returned results by supplying a base path prefix in the query. In this case only files that have a path that starts with `submission-1` will be returned.
```

Comment:
This is actually true only when changes in https://github.com/neicnordic/sensitive-data-archive/pull/1601 is merged and database is migrated. Hopefully we can finished them at the maintenance window.